mxcube3/routes/SampleCentring.py

```diff
@@ -688,7 +687,7 @@ def accept_centring():
     """
     mxcube.diffractometer.acceptCentring()
     centring_add_current_point()
-
+    global CENTRING_POINT_ID
```

Comment:
It seems like something is missing here ?

Reply:
yes, my bad